docs/content/guides/developer/getting-started/get-address.mdx

```diff
@@ -1,52 +1,170 @@
+│ vigorous-spinel       │ 0x6ebb36a3c1ab2124c082d93f60f518f70494b82d8d13c5aabb3abad6ec8cd82d │                │
+│ xenodochial-spodumene │ 0x7df01d3935b7ab3cd1da6828eed763ee2cc556670528a47baa6af8c251866e48 │                │
+│ silly-apatite         │ 0x8345a5f61dcb7027b35c53bb8a6638a00705b4cad92c61de2deb76d6b20c143d │                │
+│ reverent-coral        │ 0x9ac241b2b3cb87ecd2a58724d4d182b5cd897ad307df62be2ae84beddc9d9803 │ *              │
+╰───────────────────────┴────────────────────────────────────────────────────────────────────┴────────────────╯
+```
+
+Then, to switch to another address, use the command:
+
+```
+$ sui client switch --address <ADDRESS>
+```
+
+## Query information about an address 
+
+You can use a Sui network explorer to find more information about an address, its token balances, and the objects it owns. Popular Sui explorers include:
+
+- [SuiVision](https://suivision.xyz/)
+
+- [SuiScan](https://suiscan.xyz/mainnet/accounts)
+
+:::danger
+
+Data displayed on an explorer will differ depending on which network you created your address on. If you've followed the [Configure a Sui Client](/guides/developer/getting-started/configure-sui-client.mdx) page previously, you will most likely be using the Sui Testnet. Be sure to select the Testnet network on the Sui explorer, such as https://testnet.suivision.xyz/
 
 :::
 
-- [Slush — A Sui wallet](https://chromewebstore.google.com/detail/sui-wallet/opcgpfmipidbgpenhmajoajpbobppdil): Formerly known as Sui Wallet, Slush is a wallet from the same team that built Sui - Mysten Labs.
-- [Suiet | Sui Wallet](https://chromewebstore.google.com/detail/suiet-sui-wallet/khpkpbbcccdmmclmpigdgddabeilkdpd): The Suiet wallet for everyone, built on the Sui blockchain. 
-- [Glass wallet](https://chromewebstore.google.com/detail/glass-wallet-sui-wallet/loinekcabhlmhjjbocijdoimmejangoa): Glass Wallet is a non-custodial wallet that allows you to control your coins, keys, and NFT securely.
-- [Martian Aptos & Sui Wallet Extension](https://chromewebstore.google.com/detail/martian-aptos-sui-wallet/efbglgofoippbgcjepnhiblaibcnclgk): Martian is a self-custodial crypto wallet for Aptos and Sui.
-- [Surf Wallet](https://chromewebstore.google.com/detail/surf-wallet/emeeapjkbcbpbpgaagfchmcgglmebnen): Surf Wallet is a platform that allows for convenient and secure transfer of tokens and collection of NFTs on the blockchain.
-- Visit the [chrome web store](https://chromewebstore.google.com/search/sui%20wallet) for a complete list of available Sui wallet extensions.
\ No newline at end of file
+From the CLI, you can view all objects an address owns with the command:
+
+```
+$ sui client objects <ADDRESS>
+```
+
+## Next steps 
+
+Next, [obtain Testnet SUI tokens](/guides/developer/getting-started/get-coins.mdx) from the Testnet SUI faucet.
+
+If you already have Testnet SUI, then you're ready to [deploy Hello, world!](/guides/developer/getting-started/hello-world.mdx).
```

Comment:
Wouldn't this be "...have **the** Testnet SUI"? To be clear, I'm not advocating for it :)